activejob/lib/active_job/test_helper.rb

```diff
@@ -308,9 +304,9 @@ def clear_performed_jobs # :nodoc:
 
         def enqueued_jobs_size(only: nil) # :nodoc:
           if only
-            enqueued_jobs.select { |job| Array(only).include?(job.fetch(:job)) }.size
+            enqueued_jobs.count { |job| Array(only).include?(job.fetch(:job)) }
```

Comment:
Is this done for a perf optimization or is it critical to the code path? It would be better to not include refactoring to keep the blame clean.